src/Dialogs/Settings/Panels/NOTAMConfigPanel.cpp

```
@@ -0,0 +1,471 @@
+  char radius_format_display[32];
+  const char radius_format_edit[] = "%.0f";
+  StringFormat(radius_format_display,
+               ARRAY_SIZE(radius_format_display),
+               _("%%.0f %s"), unit_name);
+
+  const double min_search_radius_user = Units::ToUserDistance(1000.0);
+  const double max_search_radius_user =
+    Units::ToUserDistance(MAX_NOTAM_REQUEST_RADIUS_KM * 1000.0);
+  const double step_search_radius_user = Units::ToUserDistance(10000.0);
+
+  AddFloat(_("Search Radius"),
```

Cppcheck (2.20.0)

[error] Rethrowing current exception with 'throw;', it seems there is no current exception to rethrow. If there is no current exception this calls std

(rethrowNoCurrentException)